core/src/MsalIdHelper.cs

```diff
@@ -68,6 +68,10 @@ internal static class MsalIdHelper
     {
         public static IDictionary<string, string> GetMsalIdParameters()
         {
+            if (CorePlatformInformationBase.Instance == null)
+            {
+                throw CoreExceptionFactory.Instance.GetClientException("someErrorCode", "Platform Not Supported");
```

Comment:
nit: would be helpful to put the platform here as well, platform information has it.

Reply:
yeah, this whole method needs updating including the class calling it. So many re-initializations, wastes CPU at very little Memory cost. Considering whether I want to add that here or not.